File: leetcode/integer division/412. Fizz Buzz/golang/main.go
package main

import (
	"fmt"
	"strconv"
)

func fizzBuzz(n int) []string {
	ans := make([]string, 0, n)
	f, b, fb := "Fizz", "Buzz", "FizzBuzz"

	for i := 1; i <= n; i++ {
		is_f := i%3 == 0
		is_b := i%5 == 0

		if is_f && is_b {
			ans = append(ans, fb)
		} else if is_f {
			ans = append(ans, f)
		} else if is_b {
			ans = append(ans, b)
		} else {
			ans = append(ans, strconv.Itoa(i))
		}
	}
	return ans
}
func main() {
	n := 54324
	fmt.Println(fizzBuzz(n))
}

File: leetcode/integer division/412. Fizz Buzz/python/main.py
from ast import List


class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans = []
        f, b, fb = "Fizz", "Buzz", "FizzBuzz"

        for i in range(1, n + 1):
            is_f, is_b = i % 3 == 0, i % 5 == 0

            if is_f and is_b:
                ans.append(fb)
            elif is_f:
                ans.append(f)
            elif is_b:
                ans.append(b)
            else:
                ans.append(str(i))
        return ans


n = 1445
print(Solution().fizzBuzz(n))
